resources/atmLocations.ts

```diff
@@ -0,0 +1,28 @@
+import { AtmLocation } from "../types/atmLocation"
+import { UnitResponse, UnitConfig, Coordinates, Address } from "../types/common"
+import { BaseResource } from "./baseResource"
+
+export class AtmLocations extends BaseResource {
+
+    constructor(token: string, basePath: string, config?: UnitConfig) {
+        super(token, basePath + "/atm-locations", config)
+    }
+
+    public async list(params?: AtmLocationListParams): Promise<UnitResponse<AtmLocation[]>> {
+        const parameters = {
+            ...(params?.coordinates && { "filter[coordinates]": params?.coordinates }),
+            ...(params?.address && { "filter[address]": params?.address }),
+            ...(params?.postalCode && { "filter[postalCode]": params?.postalCode }),
+            ...(params?.searchRadius && { "filter[searchRadius]": params?.searchRadius })
+        }
```

Comment:
```suggestion
            ...(params?.coordinates && { "filter[coordinates]": params.coordinates }),
            ...(params?.address && { "filter[address]": params.address }),
            ...(params?.postalCode && { "filter[postalCode]": params.postalCode }),
            ...(params?.searchRadius && { "filter[searchRadius]": params.searchRadius })
        }
```

Reply:
they are already defined after the first condition, no? (no need for "?" second time)

Reply:
> they are already defined after the first condition, no? (no need for "?" second time)

you are right